klt-cert-manager/Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= klc-certmanager:latest
+IMG ?= klt-cert-manager:latest
 # RELEASE_REGISTRY is the container registry to push into.
 RELEASE_REGISTRY?=ghcr.io/keptn
 RELEASE_TIME=$(shell date +%Y%m%d%s)
```

Comment:
shouldn't this be klt-cert-manager as well ? https://github.com/keptn/lifecycle-toolkit/pull/652/files#diff-75741f05e94cf48479050ad68dd39beb9b44c249f18876b4ba010c57ae9355ccR10